docs/usage/configuration.rst

```diff
@@ -0,0 +1,312 @@
+
+    Default
+        Not set; no timeout checks are performed.
+
+    Example
+        Make sure that :literal:`foo.bar` consistently produces values:
+
+        .. code-block:: json
+
+            {
+                "checks": {
+                    "foo": {
+                        "metrics": ["foo.bar"],
+                        "timeout": "1min"
+                    }
+                }
+            }
+
+:code:`ignore` (list of numbers)
+    A list of values that are never considered to generate :literal:`WARNING` or :literal:`CRITICAL` reports.
+
+    This is intended to be used for metrics that yield spurious, but fixed values that should be ignored,
+    even if they are within an otherwise abnormal range.
+    An example would be a faulty measuring device which produces the value :literal:`0.0` on encountering in an internal error,
+    but where :literal:`warning_below = 5.0`.
+
+    .. note::
+
+        **Use with care**.
+        The implementation essentially only performs a floating-point equality test to filter values.
+
+        If this sounds like a bad idea to you, you are probably right.
+        Trust me, this is here because some source cannot be fixed easily.
+
+    Example
+        A source computing a `Power factor <https://en.wikipedia.org/wiki/Power_factor>`_
+        for an AC electrical power system reports a metric :literal:`ac_system.power_factor`.
+        If the power factor is too low, a warning should be generated.
+        It might be that the source calculates a power factor of :literal:`0.0` on low draw.
+        Since a low power factor on low draw might not be considered a problem,
+        ignore the value :literal:`0.0`:
+
+        .. code-block:: json
+
+            {
+                "checks": {
+                    "low-draw": {
+                        "metrics": ["ac_system.power_factor"],
+                        "warning_below": 0.8,
+                        "ignore": [0.0]
+                    }
+                }
+            }
+
+:code:`resend_interval` (:ref:`duration<Duration>`)
+    .. _resend_interval:
+
+    Period of time after which the current state of this check is sent again to the server, eventhough it might not have changed.
+    This is necessary since passive checks are considered to be in an :literal:`UNKNOWN` state by Centreon/Nagios
+    if they have not sent a report for a certain time.
+
+    Default
+        Inherited from the :ref:`global resend interval<global_resend_interval>`.
+
+:code:`transition_debounce_window` (:ref:`duration<Duration>`)
+    If necessary, :command:`metricq_sink_nsca` tries to reduce the number of spurious :literal:`WARNING` or :literal:`CRITICAL` reports.
+    We call this process *"transition debouncing"*.
+    It does so by keeping a history of state transitions for each metric,
+    only sending an abnormal report if the majority of recent state transitions indicate an abnormal state.
```

Comment:
I don't understand what it does.

Provide an example and/or improve the description.